addons/l10n_fr/data/account.account.template.csv

```diff
@@ -1,233 +1,180 @@
 "pcg_2772","Actions propres ou parts propres en voie d'annulation","277200","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_278"," Mali de fusion sur actifs financiers","278000","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
+"pcg_278","Mali de fusion sur actifs financiers","278000","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
 "pcg_279","Versements restant à effectuer sur titres immobilisés non libérés","279000","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2801","Amortissements des immobilisations incorporelles - Frais d'établissement (même ventilation que celle du compte 201)","280100","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2803","Amortissements des immobilisations incorporelles - Frais de recherche et de développement","280300","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2805","Amortissements des immobilisations incorporelles - Concessions et droits similaires, brevets, licences, logiciels, droits et valeurs similaires","280500","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2807","Amortissements des immobilisations incorporelles - Fonds commercial","280700","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2808","Amortissements des autres immobilisations incorporelles","280800","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_28081"," Amortissements du mali de fusion sur actifs incorporels","280810","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2812","Amortissements des immobilisations corporelles - Agencements aménagements de terrains (même ventilation que celle du compte 212)","281200","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2813","Amortissements des immobilisations corporelles - Constructions (même ventilation que celle du compte 213)","281300","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2814","Amortissements des immobilisations corporelles - Constructions sur sol d'autrui (même ventilation que celle du compte 214)","281400","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2815","Amortissements des immobilisations corporelles - Installations matériel et outillage industriels (même ventilation que celle du compte 215)","281500","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_2818","Amortissements des autres immobilisations corporelles (même ventilation que celle du compte 218)","281800","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
-"pcg_28187"," Amortissement du mali de fusion sur actifs corporels","281870","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
+"pcg_2801","Frais d'établissement (même ventilation que celle du compte 201)","280100","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
+"pcg_2803","Frais de développement","280300","asset_fixed","l10n_fr.l10n_fr_pcg_chart_template","","False"
```

Comment:
I will say that the fact that we don't make any diff in the name between 203, 2803, 2903 feels a bit weird.
Nothing really wrong, as the group defines it, I'm just making the remark